contents/blog/pick-cofounder.md

```diff
@@ -0,0 +1,62 @@
+---
+date: 2021-04-28
+title: An engineer's guide to picking a cofounder
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+---
+
+By: [Mohammed Shehu](https://twitter.com/shehuphd), Content Lead @ PostHog
+
+What could be worse than handing out 50% of your company's equity to a co-founder who doesn't understand the product as much as you do, and all they seem to bring is the ability to talk?
+
+Likewise, a sales-focused founder may be struggling to know what makes you great, or - worse - may not appreciate the importance of development. There's a case to be made for and against getting a co-founder, and that's what we'll look at in this post.
+
+Let's dive in.
+
+## Do you need a co-founder?
+
+At PostHog, Tim didn't even know he wanted to start a company, so the answer here was yes!
```

Comment:
We can link to [this piece](https://posthog.com/blog/inflated-risk-seems-riskier) for context.

Reply:
Please do! No need to ask about should I add a link - I'd encourage you to add lots of them as you see fit, especially if it's to content we've produced (on any medium - posts/social media)